app/src/androidTest/java/org/wikipedia/pages/OnboardingPage.kt

```diff
@@ -0,0 +1,17 @@
+package org.wikipedia.pages
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers
+import org.wikipedia.R
+
+class OnboardingPage {
+
```

Comment:
Sometimes there are no empty lines after class name, sometimes there are (like here). In general, try to avoid too many empty lines. Use them between methods declaration or important part of the class.